Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM node:lts-alpine
+FROM node:lts-alpine AS build-step
```

Comment:
You could specify the version here like:

```
FROM node:lts-alpine3.14 AS build-step
```
to avoid unexpected changes

Reply:
```suggestion
FROM node:lts-alpine3.14 AS build-step
```